test/tests/functional/pbs_cray_vnode_per_numa.py

```diff
@@ -159,17 +160,27 @@ def test_settings(self):
 
         # Compare the pbsnodes output when vnode_per_numa_node was unset
         # versus when vnode_per_numa_node was set to False.
-
-        if (len(vnodes_combined) == len(vnodes_combined1)):
+        # List of resources to be ignored while comparing.
+        ignr_rsc = ['license', 'last_state_change_time']
+        len_vnodes_combined1 = len(vnodes_combined1)
+        len_vnodes_combined = len(vnodes_combined)
+        if len_vnodes_combined == len_vnodes_combined1:
             self.logger.info(
                 "pbsnodes outputs are equal in length.")
-            for n in vnodes_combined:
-                if n not in vnodes_combined1:
-                    self.logger.error(
-                        "ERROR vnode %s has differing element." % n['id'])
-                    self.assertTrue(False)
-            self.logger.info(
-                "pbsnodes outputs are the same.")
+            for n in range(0, len_vnodes_combined - 1):
+                for key in vnodes_combined[n].keys():
+                    if key in ignr_rsc:
+                        continue
+                    if key in vnodes_combined1[n].keys():
+                        if n[key] != vnodes_combined1[0][key]:
+                            self.logger.error("ERROR vnode % s has \
```

Comment:
You have a space between % and s.  Also, don't put a \\ in the middle of a string.  You'll end up with all that indentation in the middle of it.  End the string with a " and then the \\ and start the next string after the indentation with a "